src/Neo.VM/Types/InteropInterface.cs

```diff
@@ -17,10 +17,12 @@ namespace Neo.VM.Types
     /// Represents an interface used to interoperate with the outside of the the VM.
     /// </summary>
     [DebuggerDisplay("Type={GetType().Name}, Value={_object}")]
-    public class InteropInterface : StackItem
+    public class InteropInterface : StackItem, IMemoryItem
     {
         private readonly object _object;
 
+        public int Size => 4;
```

Comment:
`InteropInterface` object size is hard to measure and abuse, I think we can ignore this type.

Reply:
Maybe it's better to define the size of each, because maybe some of them can be bigs
https://github.com/neo-project/neo/blob/f5e257c11c514e47cdc1da6116e475a7324ba1ac/src/Neo/SmartContract/Native/CryptoLib.BLS12_381.cs#L40